src/dpp/webhook.cpp

```diff
@@ -30,7 +30,7 @@ using json = nlohmann::json;
 
 const size_t MAX_ICON_SIZE = 256 * 1024;
 
-webhook::webhook() : managed(), type(w_incoming), guild_id(0), channel_id(0), user_id(0), application_id(0), image_data(nullptr)
+webhook::webhook() : managed(), type(w_incoming), guild_id(0), channel_id(0), application_id(0), image_data(nullptr)
```

Comment:
```diff
-image_data(nullptr)
+image_data()
```
https://github.com/brainboxdotcc/DPP/actions/runs/6592484336/job/17913192543?pr=960#step:8:13

Reply:
I just outright removed the image_data declaration, it's redundant at that point.